install_config/configuring_vsphere.adoc

```diff
@@ -157,6 +157,27 @@ determined by using the command `uname -n`. In case of a mismatch, the native
 cloud integration for those providers will not work.
 ====
 
+== Configuring OpenShift Container Platform for vSphere using Ansible
+
+You can configure OpenShift Container Platform for VMware vSphere (VCP) by modifying the Ansible inventory file at installation time or after installation.
```

Comment:
I'd s/at installation time/during installation

Reply:
You're right !